### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by ItzAron** [mame82/P4wnP1_aloa#173]
Is there a way to run a ducky script as soon as plugged in?

**Comment by 21weberer** [comment]
yes, if you power the pi with a portable power bank and connect to the access point and run a script, you can just plug the pi0 in and it will do the job. I do want to note, however, you will need to rewrite the ducky script into HIDScript, or it will not function. I would also reccomend using the line waitLED command (cant remember full syntax, might need to look around) so that the HID can initalize its keyboard.
